src/net/smooth/parser.js

```diff
@@ -69,6 +60,25 @@ const profiles = {
   ],
 };
 
+function extractCodec(fourCC, rep) {
```

Comment:
could you make the signature `(fourCC, codecPrivateData)` to indicate we have no other "dependency" on the rep object.